src/lib/bootloader/autoyast_converter.rb

```diff
@@ -27,6 +52,111 @@ def export(config)
 
     private
 
+      def import_default(data, bootloader)
+        DEFAULT_BOOLEAN_MAPPING.each do |key, method|
+          next unless data["global"][key]
+
+          bootloader.grub_default.public_send(method).value = data["global"][key] == "true"
```

Comment:
Could a malformed profile (containing a unsupported key) a `NoMethodError` in `bootloader.grub_default.public_send(method)`?